system-test/system-test.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# Stopping the script if any command fails
+set -e
+source bin/env.sh
+
+function cleanup {
+  echo "Removing pulse-system-test from tmp directory"
+  rm  -r /tmp/pulse-system-test
+  # killing log-collector, collection-roller, Alert-engine, Example application.
+  kill_services
+  echo "Sevices are terminated/killed"
+}
+
+trap cleanup EXIT
+mkdir -p /tmp/pulse-system-test
+mkdir -p system-test/log_files
+
+# Writing logs to local directory
+export collection_roller_log="system-test/log_files/collection_roller.log"
+export alert_engine_log="system-test/log_files/alert_engine.log"
+export log_collector_log="system-test/log_files/log_collector.log"
+export application_log="system-test/log_files/application.log"
+
+kill_services(){
```

Comment:
I don't think this needs to be a function, can you just put it inside 'cleanup'? If you think it needs to be a function, it might make more sense at the top of the file instead of in the middle of the script logic

Reply:
Fixed , moved the kill commands to the cleanup function.